testplan/common/utils/timing.py

```diff
@@ -320,30 +327,35 @@ def record(self, key):
             >>> with timer.record('my-key'):
             >>>  ... custom code ...
             >>>  ... custom code ...
-            >>> timer['my-key'].elapsed
+            >>> timer['my-key'][-1].elapsed
```

Comment:
What is the usecase to have more than one timer for a key? Most of the code elsewhere I see either the first item, or (most cases) the last item is used only.

If the last and the first is so significan, can those guys have some dedicated accessors

like Timer.last("key"), Timer.first("key")